### PR TITLE
[Maintenance] Add Sylius 1.12 support

### PR DESCRIPTION
**Pull request by jakubtobiasz**
This PR comes with brand new and shiny support for Sylius 1.12.

### CODE REVIEW

File: tests/Application/config/packages/assets.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+framework:
```
Comment (GSadee) [inline tests/Application/config/packages/assets.yaml:1]: Is this config needed here as it is defined in `SyliusUiExtension` or you just want it to be clear?

File: tests/Application/config/packages/webpack_encore.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+webpack_encore:
```
Comment (GSadee) [inline tests/Application/config/packages/webpack_encore.yaml:1]: Similar as above, these configs are defined in Sylius, are they needed here?

File: tests/Application/config/packages/assets.yaml (hunk 1 above)
Comment (jakubtobiasz) [reply tests/Application/config/packages/assets.yaml:1]: @GSadee PluginSkeleton uses an alpha version so I don't have those changes. It is not a blocker so I decided to leave it as it is until beta.

File: tests/Application/config/packages/webpack_encore.yaml (hunk 2 above)
Comment (jakubtobiasz) [reply tests/Application/config/packages/webpack_encore.yaml:1]: @GSadee PluginSkeleton uses an alpha version so I don't have those changes. It is not a blocker so I decided to leave it as it is until beta.

File: composer.json [hunk 3]
```diff
@@ -82,5 +83,7 @@
             "assets:install %PUBLIC_DIR%": "symfony-cmd",
             "security-checker security:check": "script"
         }
-    }
+    },
+    "minimum-stability": "dev",
```
Comment (GSadee) [inline composer.json]: As we need dev dependency only for Sylius, can you solve this similar to PayPalPlugin and use `1.12.x-dev` as `sylius/sylius`

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Jakub! :tada:

**Pull request merged**